atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -64,6 +64,7 @@
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
     is_allowed_to_fail = False
+    explicit_labels = ['release', 'version']
```

Comment:
Its still a bit different (the value of the label is not being overwritten, its just being set again for visibility), but its better than `explicit`. Updated